web-component-notification-center-example/frontend/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <!-- Web Component Bundled Version CDN -->
+    <script src="https://novu-web-component.netlify.app/index.js"></script>
+    <title>Notification Generator</title>
+</head>
+
+<body>
+    <div class="header">
+        <h1>Notifications Generator</h1>
+
+        <notification-center-component style="display: inline-flex" application-identifier="SWMw97ec1ZNA"
```

Comment:
This identifier here is a public identifier, but it means anybody that uses this and runs the trigger will send to this app. So, are you sure you want to leave this open to the public?